contracts/actions/mstable/MStableClaim.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "contracts/interfaces/mstable/IBoostedVaultWithLockup.sol";
+import "./helpers/MainnetMStableAddresses.sol";
+import "contracts/utils/TokenUtils.sol";
+import "contracts/utils/SafeMath.sol";
+import "../ActionBase.sol";
+
+contract MStableClaim is ActionBase, MainnetMStableAddresses {
+    using TokenUtils for address;
+    using SafeMath for uint256;
+
+    struct Params {
+        address vaultAddress;   // vault contract address for the imAsset (imAssetVault address)
+        address to;             // address that will receive the claimed MTA rewards
+        uint256 first;
```

Comment:
What are first and last? Maybe comments and/or better names

Reply:
First and last index in the users rewards array on which to claim.
I propose we document this in gitbook similarly how we did it for  firstHint, lastHint (Liquity).